.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+## Description
+<!-- Add a description here -->
+
+### Documentation sets edited in this PR
+
+_Check all that apply._
+
+- [ ] Stateful (`docs/en/observability/*`)
+- [ ] Serverless (`docs/en/serverless/*`)
+- [ ] Integrations Developer Guide (`docs/en/integrations/*`)
+- [ ] None of the above
+
+### Related issue
+Closes # <!-- Add the issue this PR closes here -->
+
+## Checklist
```

Comment:
Feel free to adjust. I don't know if this should be in a comment or in the text that gets posted to the PR template. On one hand it's easy to dismiss comments... on the other hand it would be repetitive and make PR descriptions so long... 

```suggestion
## Checklist

<!--
Add labels to:
1. Backport to other versions (`backport-*`):
    - `backport-8.x` to backport to the latest minor
    - `backport-skip` to not backport (for example, for serverless docs) 
    - `backport-main` to "backport" to `main` if the target branch is _not_ `main`
    - Individual `backport-*` labels to target specific minor versions
2. Surface blocking reviews (`needs-*-review`):
    - `needs-writer-review` for codeowners
    - `needs-dev-review` for dev team
    - `needs-product-review` for PM review
3. Build serverless preview docs: `ci:doc-build`
-->
```

Reply:
I think a comment makes sense. Don't want to muddy the PR description too much.

Reply:
This looks good!